src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -62,10 +62,11 @@ public function __construct()
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('vars', [
-                'type' => self::TYPE_JSON,
+                'type' => Response::MODEL_VARIABLE,
                 'description' => 'Function environment variables.',
```

Comment:
Let's decide on a word to describe these and be consistent. I say we call them `function variables` as you do in the endpoint descriptions.